stl/src/vector_algorithms.cpp

```diff
@@ -3618,6 +3619,233 @@ namespace {
             return _Last1;
         }
     }
+
+    template <class _Traits, class _Ty>
+    const void* __stdcall __std_find_end_impl(
+        const void* const _First1, const void* const _Last1, const void* const _First2, const size_t _Count2) noexcept {
+        if (_Count2 == 0) {
+            return _First1;

```

Comment:
This is bogus, but concealed by how `<algorithm>` avoids calling this for empty needles. WG21-N4993 \[alg.find.end\]/1.3:

> Let: `i` be `last1` if `[first2, last2)` is empty, or if `(last2 - first2) > (last1 - first1)` is `true`, or if there is no iterator in the range \[...\]

Reply:
Correct